MMR.Randomizer/Templates/HtmlSpoiler.cs

```diff
@@ -1,7 +1,7 @@
 ﻿// ------------------------------------------------------------------------------
 // <auto-generated>
 //     This code was generated by a tool.
-//     Runtime Version: 17.0.0.0
+//     Runtime Version: 16.0.0.0
```

Comment:
Not sure if this is an issue? If so, how would I get the newer version? Is it based on my version of Visual Studio?